pkg/models/alerting/rules/ruler.go

```diff
@@ -527,16 +545,22 @@ func (r *ThanosRuler) UpdateAlertingRules(ctx context.Context, ruleNamespace *co
 	var (
 		itemsMap  = make(map[string][]*ResourceRuleItem)
 		respItems = make([]*v2alpha1.BulkItemResponse, 0, len(ruleItems))
+		// rules updated successfully. The key is the rule name.
 		successes = make(map[string]struct{})
-		moveMap   = make(map[string][]*ResourceRuleItem)
-		delMap    = make(map[string][]*ResourceRuleItem) // duplicate rules that need to deleted in the same resource
-
+		// rules to be moved to other resources. The key is the resource name in which the rules were.
```

Comment:
// rules to be moved to other resources. The key is the resource name in which the rules were.
=>
// rules to be moved to other resources. The key is the resource name in which the rules reside.